andela/AndelaEatsApi
# #158551899 User can view, update, create About page

**What does this PR do?**
- Adds endpoints that enable the creation, updating and viewing of the about page

**Description of Task to be completed?**
- Create an About Model
- Create migrations for the newly created Model
- Create endpoint `api/v1/about/view` that enables the viewing of About Page markup using a get request.
- Create endpoint `api/v1/about/create_or_update` that enables the creation or update of About Page markup. The endpoint creates an entry into the DB if it does not exist and there after subsequent calls will keep updating that which was created. The endpoint supports `POST` and `PATCH` requests.
- Create controllers, repositories and blueprints for the endpoints.
- Add integration and unit tests for the functionality added
- Add documentation for the endpoints above

**How should this be manually tested?**
- Pull the branch `ft-about-page-158551899` and checkout to it.
- Run the command `flask db upgrade` to run new migrations against your local database
- Start the application locally by running `python run.py runserver`
- Look through the swagger documentation by navigating to `/apidocs` to see how to use the About endpoints under the `About` section.
- Create, update or View the about page

**Any background context you want to provide?**
- There was no endpoint to store About page details. 

**What are the relevant pivotal tracker stories?**
[#158551899](https://www.pivotaltracker.com/story/show/158551899)
